enclave_build/src/lib.rs

```diff
@@ -58,6 +62,9 @@ impl<'a> Docker2Eif<'a> {
         artifacts_prefix: String,
         certificate_path: &Option<String>,
         key_path: &Option<String>,
+        img_name: &Option<String>,
+        img_version: &Option<String>,
+        meta_file_path: &Option<String>,
```

Comment:
These could be put into a separate structure. This structure should implement Default so that we can get rid of the unwraps lower in the function.

Reply:
`img_name` default value depends on one other argument - `docker_image`.
As agreed offline, would be better to combine all arguments of that function to some common struct, i.e. `Docker2EifArgs`. But that should be addressed in a separate PR.